src/viewProviders/resources.ts

```diff
@@ -102,10 +117,9 @@ export class ResourceViewProvider implements vscode.TreeDataProvider<ResourceVie
   }
 }
 
-var resourceViewProvider = new ResourceViewProvider();
```

Comment:
This was the main problem with activation -- I'm mostly convinced we were hitting a race condition between extension activation and the ResourceViewProvider.getChildren() method resolving/rejecting, and it only reared its head once we started hopping to different platforms/architectures.

Didn't affect the Topics/Schemas views since they cleanly exit when no cluster resource is set during the Kafka/SR cluster-selected commands.